modules/ROOT/pages/class-loader-library-config.adoc

```diff
@@ -9,69 +9,28 @@
 //
 //
 :page-description: You can configure class loading for application to provide access to libraries and third-party APIs that are not provided by the default settings.
-:seo-title: Class loader configuration for third party APIs and Java libraries with Open Liberty
+:seo-title: Class loader configuration for third-party APIs and Java libraries with Open Liberty
 :seo-description: You can configure class loading for application to provide access to libraries and third-party APIs that are not provided by the default settings.
 :page-layout: general-reference
 :page-type: general
 = Class loader configuration
 
 By default, an application that is running on an Open Liberty server can access a set of provided APIs and its own bundled classes and Java libraries. You can configure class loading for an application to provide access to libraries and third-party APIs that are not provided by default.
 
-Each Jakarta EE application has its own class loader, which dynamically loads Java classes into the Java virtual machine. Open Liberty assumes some default settings for all Jakarta EE applications so that they can access the supported specification APIs and the Open Liberty APIs. For example, the servlet APIs are available by default if the Java Servlets feature is enabled. Each application can access these provided APIs and any internal classes and libraries that are packaged with the application. In many cases, if an application needs to reference an implementation of a library that is not provided by the default settings, you can bundle that library with the application itself.
+Each Jakarta EE application has its own class loader, which dynamically loads Java classes into the Java virtual machine. Open Liberty assumes some default settings for all Jakarta EE applications so that they can access the supported specification APIs and the Open Liberty APIs. For example, the servlet APIs are available by default if the Jakarta Servlets feature is enabled. Each application can access these provided APIs and any internal classes and libraries that are packaged with the application. In many cases, if an application needs to reference an implementation of a library that is not provided by the default settings, you can bundle that library with the application itself.
```

Comment:
```suggestion
Each Jakarta EE application has its own class loader, which dynamically loads Java classes into the Java virtual machine. Open Liberty assumes some default settings for all Jakarta EE applications so that they can access the supported specification APIs and the Open Liberty APIs. For example, the servlet APIs are available by default if the Jakarta Servlets feature is enabled. Each application can access these provided APIs and any internal classes and libraries that are packaged with the application. Often, if an application needs to reference an implementation of a library that is not provided by the default settings, you can bundle that library with the application itself.
```

Reply:
updated